parts/k8s/kubernetesmastercustomdata.yml

```diff
@@ -504,5 +504,6 @@ coreos:
 {{else}}
 runcmd:
 - set -x
-- {{GetKubernetesMasterPreprovisionYaml}}
+- . /opt/azure/containers/provision_source.sh
+- aptmarkWALinuxAgent hold{{GetKubernetesMasterPreprovisionYaml}}
```

Comment:
shouldn't hold be "hold"?

Reply:
I'm skittish about quotes in runcmd + cloud-init. Strictly speaking the `"` are unnecessary:

```
azureuser@k8s-master-21272034-0:~$ cat test.sh 
#!/bin/bash

testfunc() {
    if [[ "$1" == "hold" ]]; then
        echo "yes"
    else
        echo "no"
    fi
}

testfunc "hold"
testfunc hold
testfunc nothold
azureuser@k8s-master-21272034-0:~$ ./test.sh 
yes
yes
no
```

Reply:
Should we standardize to not using `"hold"` anywhere and prefer `hold`?

Reply:
Yeah let's standardize to `hold` and `unhold`